benchmarks/models_config/sam_fast_torch_compile_gpu_best_latency.yaml

```diff
@@ -0,0 +1,32 @@
+---
+sam_fast:
+    vit_b:
+        benchmark_engine: "ab"
+        url: https://torchserve.pytorch.org/mar_files/sam_fast_vit_b_01ec64.mar
+        workers:
+            - 4
+        batch_delay: 100
+        batch_size:
+            - 1
+        input: "./examples/large_models/segment_anything_fast/kitten.jpg"
+        requests: 1000
```

Comment:
Are 1000 requests sufficient?

Reply:
Given that the TPS for vit_h is around 4 and the TPS for vit_b is around 5.2, I think 1000 requests should be sufficient. I have tried running the benchmark with a higher number of requests but the results were the same